src/main/xar-resources/data/xquery/xquery.xml

```diff
@@ -911,15 +911,6 @@
             that extend XPath and XQuery with cross-platform functions and capabilities. (Only those
             built into eXist are listed below; see eXist-db's Package Repository for others.)</para>
 
-        <sect2 xml:id="http-client-module">
-            <title>HTTP Client</title>
-            <para>eXist-db implements the <link condition="_blank"
-                xlink:href="http://expath.org/spec/http-client">HTTP Client Module</link>, an EXPath
-                specification for interacting with HTTP servers. This is in addition to eXist-db's
-                native function libraries for performing HTTP <link
-                xlink:href="/exist/apps/fundocs/view.html?uri=http://exist-db.org/xquery/request"
-                >requests</link>.</para>
-        </sect2>
```

Comment:
The EXPath HTTP Client wasn't removed in Adam's commit - only the native HTTP client. So I'd strip out just the last sentence of this, not the entire section.

Reply:
makes sense want to use the suggestion feature to correct this? Apparently,  i can't suggest changes to my own PR